src/prefabs/structures/DateTimePicker/options.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  buttongroup,
+  color,
+  option,
+  showIf,
+  showIfTrue,
+  text,
+  ThemeColor,
+  toggle,
+  variable,
+} from '@betty-blocks/component-sdk';
+import { showOn } from '../../../utils';
+
+export const options = {
+  actionVariableId: option('ACTION_JS_VARIABLE', {
+    label: 'Name',
+    value: '',
+    configuration: { condition: showIf('actionVariableId', 'EQ', 'never') },
+  }),
+
+  actionProperty: option('ACTION_JS_PROPERTY', {
+    label: 'Property',
+    value: '',
+  }),
+
+  label: variable('Label', { value: [''] }),
+  value: variable('Value', { value: [''] }),
+
+  validationOptions: toggle('Validation options'),
+
+  required: toggle('Required', showOn('validationOptions')),
+
+  validationValueMissing: variable('Value required message', {
+    value: ['This field is required'],
+    ...showOn('required'),
+  }),
+
+  type: text('Type', {
+    value: 'datetime',
+    configuration: {
+      condition: showIf('type', 'EQ', false),
+    },
+  }),
+
+  disableToolbar: toggle('Disable Toolbar', { value: false }),
+
+  timeFormat: text('Format', {
+    value: 'MM/dd/yyyy HH:mm:ss',
+    configuration: { placeholder: 'dd/MM/yyyy HH:mm:ss' },
+  }),
+
+  use24HourClockTime: toggle('Use 24-hour format', { value: true }),
+
+  error: toggle('Error', { value: false }),
+
+  disabled: toggle('Disabled', { value: false }),
+
+  autoComplete: toggle('Autocomplete', { value: true }),
+
+  placeholder: variable('Placeholder', { value: [] }),
+
+  helperText: variable('Helper text', { value: [] }),
+
+  variant: buttongroup(
+    'Variant',
+    [
+      ['Dialog', 'dialog'],
+      ['Inline', 'inline'],
+      ['Static', 'static'],
+    ],
+    { value: 'inline' },
+  ),
+
+  clearable: toggle('Clearable', {
+    value: false,
+    configuration: { condition: showIf('variant', 'EQ', 'dialog') },
+  }),
+
+  inputvariant: buttongroup(
+    'Input Variant',
+    [
+      ['Standard', 'standard'],
+      ['Outlined', 'outlined'],
+      ['Filled', 'filled'],
+    ],
+    { value: 'outlined' },
+  ),
+
+  fullWidth: toggle('Full width', { value: true }),
+
+  size: buttongroup(
+    'Size',
+    [
+      ['Medium', 'medium'],
+      ['Small', 'small'],
+    ],
+    { value: 'medium' },
+  ),
+
+  margin: buttongroup(
+    'Margin',
+    [
+      ['None', 'none'],
+      ['Dense', 'dense'],
+      ['Normal', 'normal'],
+    ],
+    { value: 'normal' },
+  ),
+
+  styles: toggle('Styles', { value: false }),
```

Comment:
In other components we split up the advanced / styles / validation files separately to reduce the size of this file. Might want to do the same here.